eng/Version.Details.xml

```diff
@@ -25,5 +25,29 @@
       <Uri>https://github.com/microsoft/testanywhere</Uri>
       <Sha>aa2fcc8616d988b234bc1d218465b20c56d0b82f</Sha>
     </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.diagnostics" Version="9.0.0-preview.24566.1">
+      <Uri>https://github.com/dotnet/diagnostics</Uri>
+      <Sha>8c505ca6921b5f7e9b8acc234cc8f15035537ee4</Sha>
+      <SourceBuild RepoName="diagnostics" ManagedOnly="true" />
+    </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-externals" Version="9.0.0-alpha.1.24175.1">
+      <Uri>https://github.com/dotnet/source-build-externals</Uri>
+      <Sha>4df883d781a4290873b3b968afc0ff0df7132507</Sha>
+      <SourceBuild RepoName="source-build-externals" ManagedOnly="true" />
+    </Dependency>
+    <!-- Intermediate is necessary for source build. -->
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.source-build-reference-packages" Version="10.0.0-alpha.1.24558.1">
+      <Uri>https://github.com/dotnet/source-build-reference-packages</Uri>
+      <Sha>1ebd9ce245112164207d961c0d2faea741c7c489</Sha>
+      <SourceBuild RepoName="source-build-reference-packages" ManagedOnly="true" />
+    </Dependency>
+    <!-- Intermediate is necessary for source build. -->    
+    <Dependency Name="Microsoft.SourceBuild.Intermediate.arcade" Version="10.0.0-beta.24604.4">    
+      <Uri>https://github.com/dotnet/arcade</Uri>    
+      <Sha>45d845e04c05fbe5da9838c454bbc3af1df6be81</Sha>    
+      <SourceBuild RepoName="arcade" ManagedOnly="true" />    
+    </Dependency>
```

Comment:
As discussed offline, I would like to preserve this repo using NET 9, I need to change our subscription and unblock the arcade flow. Shall we downgrade these 2 to 9.X?

Reply:
how do you want to do this, I don't think you discussed this with me :)

Reply:
Let's keep as-is for now and I'll handle when I try to downgrade

Reply:
What is the reason to downgrade?  I ask to see if this is going to flow into a 9.0 SDK or if it is for other reasons?